test/tests/kernels/hfem/array_dirichlet.i

```diff
@@ -0,0 +1,110 @@
+[Mesh]
+  [square]
+    type = GeneratedMeshGenerator
+    nx = 3
+    ny = 3
+    dim = 2
+  []
+  build_all_side_lowerd_mesh = true
+[]
+
+[Variables]
+  [u]
+    order = THIRD
+    family = MONOMIAL
+    block = 0
+    components = 2
+  []
+  [lambda]
+    order = CONSTANT
+    family = MONOMIAL
+    block = INTERNAL_SIDE_LOWERD_SUBDOMAIN
+    components = 2
+  []
+  [lambdab]
+    order = CONSTANT
+    family = MONOMIAL
+    block = BOUNDARY_SIDE_LOWERD_SUBDOMAIN
+    components = 2
+  []
+[]
+
+[AuxVariables]
+  [v]
+    order = CONSTANT
+    family = MONOMIAL
+    block = 0
+    initial_condition = '1'
+  []
+[]
+
+[Kernels]
+  [diff]
+    type = ArrayDiffusion
+    variable = u
+    block = 0
+    diffusion_coefficient = dc
+  []
+  [source]
+    type = ArrayCoupledForce
+    variable = u
+    v = v
+    coef = '1 2'
+    block = 0
+  []
+[]
+
+[DGKernels]
+  [surface]
+    type = ArrayHFEMDiffusion
+    variable = u
+    lowerd_variable = lambda
+  []
+[]
+
+[BCs]
+  [all]
+    type = ArrayHFEMDirichletBC
+    boundary = 'left right top bottom'
+    variable = u
+    lowerd_variable = lambdab
```

Comment:
replace this with `value = '0 0'`.

Reply:
```
[BCs]
  [all]
    type = ArrayHFEMDirichletBC
    boundary = 'left right top bottom'
    variable = u
    value = '0 0'
  []
[]
```
With this modification, the test failed with the following error:

tests/kernels/hfem/array_dirichlet.i:66: missing required parameter 'BCs/all/lowerd_variable'
        Doc String: "Lagrange multiplier"

Reply:
oh, sorry. My bad. This is indeed needed and we need this variable as well.